tests/test_manual.py

```diff
@@ -68,15 +67,14 @@ async def test_state_frame_number_overflow(state, session):
 
 
 @pytest.mark.asyncio
-async def test_zone_discovery(state, session):
+async def test_zone_discovery(state, session, caplog):
```

Comment:
Since `caplog` is not used in the function body, I presume you added it only for your local debugging, right? If this is the case, can you please remove it again? Everyone should be able to specify that using pytest cmdline args.

Reply:
ok i will remove it, but this is standart predefined fixture in async pytest https://docs.pytest.org/en/latest/how-to/logging.html#caplog-fixture